server/channels/api4/apitestlib.go

```diff
@@ -240,12 +246,24 @@ func SetupEnterprise(tb testing.TB, options ...app.Option) *TestHelper {
 		tb.SkipNow()
 	}
 
+	// Prevent static asset rewriting during tests by setting BuildNumber to "dev"
+	var oldBuildNumber = model.BuildNumber
+	model.BuildNumber = "dev"
+	tb.Cleanup(func() {
+		model.BuildNumber = oldBuildNumber
+	})
+
+	removeErrors := func(config *model.Config) {
+		*config.ServiceSettings.SiteURL = "http://localhost:8065"
+	}
```

Comment:
Also added this, which removed two errors that were not actually errors, just noise. I wonder if it will mess up CI. Let's experiment...